packages/nvidia_nat_semantic_kernel/tests/test_llm_sk.py

```
@@ -63,6 +61,61 @@ async def test_responses_api_blocked(self, mock_sk, oa_cfg_responses, mock_build
                 pass
         mock_sk.assert_not_called()
 
+    @pytest.mark.parametrize("verify_ssl", [True, False], ids=["verify_ssl_true", "verify_ssl_false"])
+    @patch("openai.AsyncOpenAI")
+    @patch("semantic_kernel.connectors.ai.open_ai.OpenAIChatCompletion")
+    async def test_verify_ssl_passed_to_client(self,
+                                               mock_sk,
```

Ruff (0.15.5)

[warning] Unused method argument: `mock_sk`

(ARG002)